presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java

```diff
@@ -120,73 +123,101 @@ public MongoSession(TypeManager typeManager, MongoClient client, MongoClientConf
         this.tableCache = CacheBuilder.newBuilder()
                 .expireAfterWrite(1, HOURS)  // TODO: Configure
                 .refreshAfterWrite(1, MINUTES)
-                .build(CacheLoader.from(this::loadTableSchema));
+                .build(CacheLoader.from(this::loadTable));
     }
 
     public void shutdown()
     {
         client.close();
     }
 
+    /**
+     * @return lower-case schema names
+     */
```

Comment:
Remove this comment to be consistent with other methods (getAllTables is also the same)